### PR TITLE
Add typings for simplesmtp.

### PR DESCRIPTION
**Pull request by rogierschouten**

### CODE REVIEW

File: simplesmtp/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,155 @@
+﻿// Type definitions for simplesmtp 0.3.35
+// Project: https://github.com/andris9/simplesmtp
+// Definitions by: Rogier Schouten <https://github.com/rogierschouten/>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare module "simplesmtp" {
```
Comment (mhegazy) [inline simplesmtp/index.d.ts]: do not use the external module declaration, just remove `declare module "simplesmtp" {` and everything should still work.

**Review by mhegazy — CHANGES_REQUESTED** [review]

**Comment by rogierschouten** [comment]
@mhegazy  done!

**Review by mhegazy — APPROVED** [review]
Thanks!

**Comment by mhegazy** [comment]
I have a fix for the failing CI tests in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/11835

**Pull request merged**